packages/design-system/src/components/UsaBanner/UsaBanner.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from 'react';
+import { Button } from '../Button';
+import classNames from 'classnames';
+import uniqueId from 'lodash/uniqueId';
+import { LockCircleIcon, LockIcon, UsaFlagIcon, BuildingCircleIcon } from '../Icons';
+import EnglishTranslations from '../../locale/en.json';
+import SpanishTranslations from '../../locale/es.json';
+
+type LocaleLanguage = 'en' | 'es';
```

Comment:
Should this be exported so it could potentially be used in consuming applications?